spacy/tests/test_misc.py

```diff
@@ -11,14 +11,15 @@
 from spacy.util import to_ternary_int
 from thinc.api import Config, Optimizer, ConfigValidationError
 from thinc.api import set_current_ops
+from thinc.compat import has_cupy_gpu, has_torch_mps_gpu
 from spacy.training.batchers import minibatch_by_words
 from spacy.lang.en import English
 from spacy.lang.nl import Dutch
 from spacy.language import DEFAULT_CONFIG_PATH
 from spacy.schemas import ConfigSchemaTraining, TokenPattern, TokenPatternSchema
 from pydantic import ValidationError
 
-from thinc.api import get_current_ops, NumpyOps, CupyOps
+from thinc.api import get_current_ops, NumpyOps, CupyOps, MPSOps
```

Comment:
This is not this PR's fault, but can you rearrange the `thinc` imports to be together? We can probably move `set_current_ops` to this list so we cut down one line.

Reply:
Fixed.